starwars/src/components/StarCharacter.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import StarDetails from './StarDetails.js'
+import './StarWars.css';
+

```

Comment:
It may be unnecessary to include lines 10 and 15. Passing down the charInfo array includes a *name* property for each character object.